src/django/api/processing.py

```diff
@@ -33,7 +36,53 @@ def _report_error_to_rollbar(file, request):
                 'file_name': file.name})
 
 
-def get_excel_sheet(file, request):
+def get_xlsx_sheet(file, request):
+    import defusedxml
+    from defusedxml.common import EntitiesForbidden
+
+    defusedxml.defuse_stdlib()
+
+    try:
+        wb = load_workbook(filename=file)
+        ws = wb[wb.sheetnames[0]]
+
+        return ws
+
+    except EntitiesForbidden:
+        _report_error_to_rollbar(file, request)
+        raise ValidationError('This file may be damaged and '
+                              'cannot be processed safely')
+
+
+def parse_xlsx(file, request):
+    try:
+        ws = get_xlsx_sheet(file, request)
+
+        percent_col = []
+
+        for cell in ws[2]:
+            if '%' in cell.number_format:
+                percent_col.append(get_column_letter(cell.column))
+
+        if percent_col:
+            for col in percent_col:
+                for cell in ws[col]:
+                    if cell.row != 1:
+                        cell.value = f"{cell.value * 100}%"
```

Comment:
Here is a potential implementation for a formatting helper function 

```python
def format_percent(value):
    if value <= 1.0:
        str_value = str(value * 100)
    else:
        str_value = str(value)
    if str_value[-2:] == '.0':
         str_value = str_value[0:len(str_value)-2]
    return str_value + '%'
```

Reply:
This is a nice implementation!